src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1297,39 +1316,40 @@ $$
 \epsilon_n \sim \textsf{normal}(0,\sigma).
 $$
 
-If either vector $x$ or $y$ has very large or very small values or if the
+If vector $x$ has very large or very small values or if the
```

Comment:
I would say if they have a mean that is far away from 0 or if they have a scale that's far away from 1.  The problem is that a bunch of standard normal variates can include very small values.  The problem is when all the values are small, like measuring human height in km, or when they're all really big, like measuring height in mm.